test.js

```diff
@@ -1,13 +1,13 @@
-var test = require('random-access-test')
```

Comment:
I think I was using the `standard` CLI for styling and linting.

Would you mind running `standard --fix` on the code, or maybe adding it as a dev dependency and having it run in `npm test`?

Reply:
Yep, done!